File: C++/1038.cpp
#include <iostream>
#include <map>
using namespace std;

int main()
{
    int x, y;
    map<int, float> menu;

    menu = {
        {1, 4.00},
        {2, 4.50},
        {3, 5.00},
        {4, 2.00},
        {5, 1.50}};
    cin >> x >> y;
    cout.precision(2);
    cout << fixed << "Total: R$ " << menu[x] * y << endl;
    return 0;
}

File: JavaScript/1038.js
var input = require("fs").readFileSync("/dev/stdin", "utf8");

var lines = input.split(" ");
var item = parseInt(lines[0]);
var qty = parseInt(lines[1]);

var menu = {
  1: 4.0,
  2: 4.5,
  3: 5.0,
  4: 2.0,
  5: 1.5,
};

console.log("Total: R$", (menu[item] * qty).toFixed(2));
